vignettes/ORFikOverview.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "ORFik Overview"
+author: "Håkon Tjeldnes"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{ORFik Overview}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Welcome to the `ORFik` package. This vignette will walk you through our main 
+package usage with two examples, one easy and one real world example.  You will learn how to interpret 
```

Comment:
Welcome to the `ORFik` package. This vignette will walk you through our main package usage with examples.  `ORFik` is an R package containing various functions for analysis of Ribo-Seq and RNA-Seq data. `ORFik` currently supports:

- Finding Open Reading Frames (very fast) in the genome of interest or on the set of transcripts.
- Metaplots for Ribo-Seq allowing to spot the shift.
- Shifting functions for the Ribo-Seq data.
- Various measurements of gene identity eg. FLOSS, coverage, ORFscore, entropy.

Remove what you have on learning and reinventing the wheel.